### PR TITLE
[prometheus-node-exporter] Ability to provide imagePullSecrets

### PR DESCRIPTION
**Pull request by julb**
#### What this PR does / why we need it:

Ability to parameter the imagePullSecrets for the chart when pulling image from a private registry.
For backwards compat, there is no imagePullSecrets specified.

#### Checklist
- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-node-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.0.1
 description: A Helm chart for prometheus node-exporter
 name: prometheus-node-exporter
-version: 1.15.0
+version: 1.15.1
```
Comment (monotek) [inline charts/prometheus-node-exporter/Chart.yaml:5]: New feature should raise minor version.

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-node-exporter/Chart.yaml (hunk 1 above)
Comment (julb) [reply charts/prometheus-node-exporter/Chart.yaml]: Done.

**Comment by julb** [comment]
@monotek Thanks for your review. I've done the changes.

**Review by monotek — DISMISSED** [review]

**Comment by monotek** [comment]
@prometheus-community/helm-charts 

Please review / merge :)

**Review by bismarck — CHANGES_REQUESTED** [review]
You can already set imagePullSecrets in the service account: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/templates/serviceaccount.yaml

**Pull request closed without merging**